Scripts/install_swiftlint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Installs the SwiftLint package.
+# Tries to get the precompiled .pkg file from Github, but if that
+# fails just recompiles from source.
+
+set -e
+
+SWIFTLINT_PKG_PATH="/tmp/SwiftLint.pkg"
+SWIFTLINT_PKG_URL="https://github.com/realm/SwiftLint/releases/download/0.9.1/SwiftLint.pkg"
```

Comment:
We're at SwiftLint 0.14 now. instead of having to update the number every time a new version of SwiftLint is released I think GitHub exposes an URL for the latest release, maybe we can use that